tests/AssignProjectFeatureTest.php

```diff
@@ -360,9 +360,6 @@ public function notAllowedAddFeaturesRoles(): array
             'share' => [
                 ProjectRole::SHARE,
             ],
-            'read only' => [
```

Comment:
Este som si uvedomil, ze sa meni logika toho co pania read only takze pak ked budem zakladat novy projket tak to nezalozi s povolenou RO rolou ostatne role stale testujem